site2/docs/helm-deploy.md

```diff
@@ -350,6 +350,9 @@ helm upgrade --install pulsar apache/pulsar \
     --timeout 10m \
     --set [your configuration options]
 ```
+> **Note**
+> 
+> For the first deployment, add `--set initialize=true` option to initialize bookie and pulsar cluster meta.
```

Comment:
```suggestion
> For the first deployment, add `--set initialize=true` option to initialize bookie and Pulsar cluster metadata.
```
same comments on the following contents.

Reply:
Updated.